content/providers/quick-start.json

```
@@ -766,11 +776,11 @@
     "image": null,
     "description": "Time dependent open quantum systems solver.",
     "installation": [
+      "pip install qiskit",
       "git clone https://github.com/0tt3r/QuaC-qiskit", 
-      "cd QuaC-qiskit",
-      "pip install"
+      "cd QuaC-qiskit"
     ],
-    "websiteCta": {
+     "websiteCta": {
```

It seems like this change was unintended:

```suggestion
    "websiteCta": {
```